mongodb_consistent_backup/Upload/S3/S3.py

```diff
@@ -1,6 +1,7 @@
 import os
 import logging
 
+import boto
```

Comment:
Makes sense @rob256. My only suggestion is to import boto.s3.multipart only, then.